pybedtools/bedtool.py

```diff
@@ -617,9 +618,29 @@ def tabix_intervals(self, interval_or_string):
         # tabix expects 1-based coords, but BEDTools works with
         # zero-based. pybedtools and pysam also work with zero-based. So we can
         # pass zero-based directly to the pysam tabix interface.
-        interval = helpers.string_to_interval(interval_or_string)
         tbx = pysam.TabixFile(self.fn)
-        results = tbx.fetch(str(interval.chrom), interval.start, interval.stop)
+
+        coord_re = re.compile(r"(.+):(\d+)-(\d+)?", re.VERBOSE)
```

Comment:
Calling `tabix_intervals` many times would result in lots of unnecessary compilations of the regex, but it really only has to happen once. Can you move this regex to the module level, or use the existing `helpers.coord_re`?